src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.ts

```diff
@@ -50,14 +88,21 @@ export class MissionRuntimeCreateappStepComponent extends LauncherStep implement
     });
   }
 
-  // Accessors
+  updateSelection(): void {
+    const cluster = this.getSelectedCluster();
```

Comment:
don't need this anymore the broadcaster give you the cluster right?

Reply:
I prefer using this since it can get called from different places